packages/apm/src/integrations/tracing.ts

```diff
@@ -355,6 +370,18 @@ export class Tracing implements Integration {
       };
     }
 
+    logger.log(`[Tracing] pushActivity: ${name}#${Tracing._currentIndex}`);
+    logger.log('[Tracing] activies count', Object.keys(Tracing._activities).length);
+    if (options && options.autoPopAfter) {
```

Comment:
How about `options && isNumber(options.autoPopAfter)` so it'll be possible to pass `0` which will autopop all activities. Just for debugging purposes, because why not. Right now `0` wont work as its falsy.